library/Fission/CLI/Auth.hs

```diff
@@ -44,6 +43,6 @@ withAuth action = get >>= \case
   Left err -> do
     logError $ displayShow err
     liftIO $ ANSI.setSGR [ANSI.SetColor ANSI.Foreground ANSI.Vivid ANSI.Red]
-    UTF8.putText $ Emoji.prohibited <> " Unable to read credentials. Try logging in with "
+    UTF8.putText "🚫 Unable to read credentials. Try logging in with "
```

Comment:
Turns out that the issue was emacs, not Haskell 😅 Got that fixed, and deleted the `Fission.Emoji` module.